charms/kfp-viz/tests/unit/test_operator.py

```diff
@@ -15,83 +14,117 @@
 # TODO: Tests missing for config_changed and dropped/reloaded relations and relations where this
```

Comment:
I think this makes sense, will you add them in this PR?
You could have something like [this](https://github.com/canonical/kfp-operators/blob/kf-4073-sidecar-rewrite-base-charm/charms/kfp-schedwf/tests/unit/test_operator.py#L65)